src/main/java/io/openmanufacturing/ame/Application.java

```diff
@@ -13,12 +13,31 @@
 
 package io.openmanufacturing.ame;
 
+import java.lang.reflect.Field;
+
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.util.ReflectionUtils;
+
+import io.openmanufacturing.sds.aspectmodel.shacl.constraint.JsConstraint;
+import io.openmanufacturing.sds.aspectmodel.versionupdate.MigratorService;
+import io.openmanufacturing.sds.aspectmodel.versionupdate.MigratorServiceLoader;
 
 @SpringBootApplication
 public class Application {
    public static void main( final String[] args ) {
+      // Since we have no rights with the local service in the class path we have to use the MigratorService ourselves.
+      final MigratorServiceLoader migratorService = new MigratorServiceLoader();
```

Comment:
Such workarounds should not be necesary for other ESMF modules. Please consider opening an issue to change the MigratorServiceLoader instead.

Reply:
Fixed